incognia_test.go

```diff
@@ -546,9 +624,19 @@ func (suite *IncogniaTestSuite) mockPostTransactionsEndpoint(expectedToken strin
 			return
 		}
 
+		requestQueryString := r.URL.Query()
+		requestEvalQueryString := requestQueryString["eval"]
+		suite.Equal(expectedQueryString["eval"], requestEvalQueryString)
```

Comment:
Is the goal here to do a shallow comparison of attributes? If so, we could make this a bit more extensible:
```suggestion
		requestQueryString := r.URL.Query()
		for parameters := range requestQueryString {
                    suite.Equal(expectedQueryString[parameter], requestQueryString[parameter])
                }
```